### ISSUE TITLE
Update to mysql 8

### ISSUE THREAD

**Issue by pieterlukasse** [thehyve/cbioportal-docker#58]
Would be nice to update from Mysql 5.7 to Mysql 8 when updating to the next cBioPortal release (3.0.0).

**Comment by fedde-s** [comment]
I agree; I saw commits addressing the incompatibility in cBioPortal v2.1.1.
